assemblyline/common/tagging.py

```diff
@@ -15,13 +15,21 @@ def tag_list_to_dict(tag_list: list[dict]) -> dict:
     return tag_dict
 
 
-def tag_dict_to_list(tag_dict: dict, safelisted: bool = False) -> list[dict]:
-    return [
-        {'safelisted': safelisted, 'type': k, 'value': t, 'short_type': k.rsplit(".", 1)[-1]}
-        for k, v in tag_dict.items()
-        if v is not None
-        for t in v
-    ]
+def tag_dict_to_list(tag_dict: dict, safelisted: bool = False, ai: bool = False) -> list[dict]:
+    if ai:
+        return [
+            {'type': k, 'value': t}
+            for k, v in tag_dict.items()
+            if v is not None
+            for t in v
+        ]
+    else:
+        return [
+            {'safelisted': safelisted, 'type': k, 'value': t, 'short_type': k.rsplit(".", 1)[-1]}
+            for k, v in tag_dict.items()
+            if v is not None
+            for t in v
+        ]
 
```

Comment:
if you're going to have a function that basically does two disjoint things based on a hardcoded bool just make it two functions.